client/ratings_reviews/components/ReviewsList.jsx

```diff
@@ -1,11 +1,76 @@
 import React from 'react';
+import $ from 'jquery';
 import ReviewTile from './ReviewTile.jsx';
 
-const ReviewsList = (props) => (
-  <div>
-    reviewslist
-    <ReviewTile />
-  </div>
-);
+class ReviewsList extends React.Component {
+  constructor(props) {
+    super(props);
+
+    $.ajax({
+      url: `reviews/?product_id=${this.props.product_id}&page=1&count=100&sort=relevant`,
```

Comment:
What if there are more reviews than 100? You might want to consider fetching fewer reviews from the server and somehow checking to see if there are more left in addition to the ones you have.

Reply:
@matthewwrobel I put the count (100) as a placeholder when I first figuring the API call. Completely forgot to change this part. You made a fantastic point about checking if there are more reviews left, I will get to it!